contractsV5/bridges/bancor/KyberBancorReserve.sol

```diff
@@ -148,6 +153,26 @@ contract KyberBancorReserve is IKyberReserve, Withdrawable, Utils {
         return true;
     }
 
+    event NewPathsSet(IERC20[] ethToBntPath, IERC20[] bntToEthPath);
+
+    function setNewEthBntPath(IERC20[] memory _ethToBntPath, IERC20[] memory _bntToEthPath) public onlyAdmin {
+        require(_ethToBntPath.length != 0, "setNewEthBntPath: path should have some elements");
+        require(_bntToEthPath.length != 0, "setNewEthBntPath: path should have some elements");
+        if (address(bancorNetwork) != address(0)) {
```

Comment:
can it be zero?

Reply:
with our logic it shouldn't be zero, I removed check here